config/demo

```diff
@@ -2,13 +2,13 @@ include Makefile
 
 export instanceid = ui
 export base_url = /
-export api_url = https://api.camptocamp.org
-export api_url_internal = https://api.camptocamp.org
-export api_url_host = api.camptocamp.org
+export api_url = http://api.demov6.camptocamp.org
```

Comment:
Why switching to http? The demo should use https URLs. Ditto for the discourse URL below.
Anyway this file no longer used when updating the demo, everything is done using env vars set in the ``docker-compose.yml`` file.